__tests__/app.test.js

```diff
@@ -13,3 +26,22 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topics, each with slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+       console.log(topics)
+        expect(topics).toHaveLength(3)
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject(
+           {slug: expect.any(String),
+            description: expect.any(String),
+            }
+          )
+        })
+      })
+  })
+})
```

Comment:
How could we handle a user trying to access an endpoint that doesn't exist? It would be great to have a test for this and set up this functionality when you can :)